pom.xml

```diff
@@ -55,7 +55,7 @@ SPDX-License-Identifier: AGPL-3.0-only
                 </inputSpec>
                 <modelPackage>com.zextras.carbonio.user_management.generated.model</modelPackage>
                 <output>./</output>
-
+                <modelNameSuffix>dto</modelNameSuffix>
```

Comment:
This change on the generation configuration must be removed for a lot of reason, besides the evaluation on the utility of that on models, this is a global setting and it's changing also models already present in the project. That's really bad since you're touching code that should not related to this task